config/routes.rb

```diff
@@ -68,6 +68,12 @@
       resources :achievements
       get 'settings' => 'course_settings#index', as: :settings
       patch 'settings' => 'course_settings#update', as: :update_settings
+
+      resources :registrations, only: [:create, :index, :destroy, :update] do
+        collection do
+          post 'destroy_multiple', as: :destroy_multiple
```

Comment:
If this is bad we may want to disallow batch update for non-javascript version and only handle batch request using javascript. That way, the controller will be shorter, too.
> On Apr 26, 2015, at 12:31 PM, Joel Low <notifications@github.com> wrote:
> 
> In config/routes.rb <https://github.com/Coursemology/coursemology2/pull/142#discussion_r29106656>:
> 
> > @@ -68,6 +68,12 @@
> >        resources :achievements
> >        get 'settings' => 'course_settings#index', as: :settings
> >        patch 'settings' => 'course_settings#update', as: :update_settings
> > +
> > +      resources :registrations, only: [:create, :index, :destroy, :update] do
> > +        collection do
> > +          post 'destroy_multiple', as: :destroy_multiple
> ...and yet I can' think of a correct solution.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/Coursemology/coursemology2/pull/142/files#r29106656>.
>